json-schemas/interface-methods/events-query.json

```diff
@@ -37,7 +38,6 @@
         },
         "filters": {
           "type": "array",
-          "minItems": 1,
```

Comment:
Unrelated to this PR, but didn't like the duality between an undefined filter and empty array, so made filters required and allowed an empty array in the message schema.